public/sass/components/_form_select_box.scss

```diff
@@ -172,18 +173,22 @@ $select-input-bg-disabled: $input-bg-disabled;
   margin-right: 10px;
 }
 
-.gf-form-select-box__option-group__header {
-  display: flex;
-  align-items: center;
-  justify-content: flex-start;
-  justify-items: center;
-  cursor: pointer;
-  padding: 7px 10px;
-  width: 100%;
-  border-bottom: 1px solid $hr-border-color;
-  text-transform: capitalize;
+.gf-form-select-box__option-group {
+  max-width: 600px;
```

Comment:
don't nest these css classes

Reply:
yep! fix is in the react-picker branch. I'll merge that into this branch